.travis.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     before_install:
     - docker pull centos:centos7
     script:
+    - ./include/travis/run_in_centos7_docker.sh make rpmlint
```

Comment:
I thinik you should be able to chain these together to run in the same docker instance:

```
- ./include/travis/run_in_centos7_docker.sh make rpmlint && ./include/travis/run_in_centos7_docker.sh include/travis/mock_build_test.sh
```

Pity that the native linux releaes in Travis is only the Trusty release of Ubuntu.  _rpmlint_ was added in _Xenial_, which is the next LTS after _Trusty_.

Reply:
This won't work, as make isn't installed in the container.

We should probably move some of the packages installed in ```include/travis/mock_build_test.sh``` into ```./include/travis/run_in_centos7_docker.sh``` so we have a basic environment to run things like rpmlint.

Reply:
So with https://github.com/intel-hpdd/lustre-ldiskfs-zfs/pull/1/commits/d099bac0d2c93519ecf6e6d5f9adbb5cb594af65, you could now chain these together to run in the same container, yes?
Or even just add the `make rpmlint` to the `mock_build_test.sh`.

Reply:
Yes. I think adding make rpmlint to the test is the way to go.